analyzer/src/test/java/com/novoda/sqlite/StringUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.novoda.sqlite;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StringUtilTest {
+    @Test
+    public void shouldCamelizeColumnNames(){
+        assertEquals("ColumnId", StringUtil.camelify("COLUMN_ID"));
+        assertEquals("Columnid", StringUtil.camelify("columnID"));
+        assertEquals("Columnid", StringUtil.camelify("COLUMNID"));
```

Comment:
ah current state right. 3 tests would make more sense though, one for each input you change - this allows fail fast and to see all test failures (rather than test - break 1st assert - push fix - test - break 2nd assert